lock/src/main/java/org/infinispan/lock/api/impl/entries/ClusteredLockValue.java

```diff
@@ -0,0 +1,104 @@
+package org.infinispan.lock.api.impl.entries;
+
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+import java.util.Collections;
+import java.util.Objects;
+import java.util.Set;
+
+import org.infinispan.commons.marshall.AdvancedExternalizer;
+import org.infinispan.commons.marshall.MarshallUtil;
+import org.infinispan.lock.api.impl.externalizers.ExternalizerIds;
+
+/**
+ * Lock object inside the cache. Holds the lock owner, the lock request id and the status of the lock.
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 9.2
+ */
+public class ClusteredLockValue {
+
+   public static final AdvancedExternalizer<ClusteredLockValue> EXTERNALIZER = new Externalizer();
+   private String requestId;
```

Comment:
This is a protection for unlocks that is not used still because the problem is more something needed on reentrant locks. But I left it there because it helps for debug the different requests done by the same node.
The problem I imagined was : a node acquieres the lock. the node leaves. the unlock for that node is raised. meanwhile the node is back and reacquires the lock. the unlock happens. In case of reentrant locks, if the request that actually holds the lock is not the one that was supposed to be unlocked, we should not unlock. But this is not an issue for non reentrant locks, because if the unlock did not happen, and the node tries to lock, it won't be able to lock and will bloc.